unittests/files/opt/seagate/sspl/setup/resource_map/test_storage.py

```diff
@@ -174,5 +254,24 @@ def test_get_platform_sensors_empty(self, encl_response):
         assert "voltage" not in resp
 
 
+    @patch(
+        "files.opt.seagate.sspl.setup.resource_map.storage.StorageMap.get_realstor_encl_data"
+    )
+    def test_get_fanmodules_info(self, encl_response):
+        encl_response.return_value = json.loads(
+            FANMODULES_RESPONSE
+        )['api-response']['fan-modules']
+        resp = self.storage_map.get_fanmodules_info()
+        assert resp[0]['uid'] == 'fan_module_0.0'
+        assert resp[0]['health']['status'] == 'OK'
+        assert resp[0]['health']['description'] == 'FAN is in good health'
```

Comment:
So if the health of the Fan is bad then this unit test will fail, is that intended?